### DISCUSSION TITLE
Invert behavior of name[missing] for user-specified modules (e.g., for import_tasks)

### DISCUSSION THREAD

**Discussion by librick** [ansible/ansible-lint#3438] [Ideas]
When using the `import_tasks` module, I think that requiring a name field above every `ansible.builtin.import_tasks: <sometaskfile.yml>` is gratuitous. E.g., I commonly have a file that includes multiple import_tasks lines:
```
---
- ansible.builtin.import_tasks: my_first_tasks.yml
- ansible.builtin.import_tasks: my_second_tasks.yml
- ansible.builtin.import_tasks: my_third_tasks.yml
```
and I see messages such as:
```
name[missing]: All tasks should be named.
main.yml:2 Task/Handler: import_tasks my_first_tasks.yml
name[missing]: All tasks should be named.
main.yml:3 Task/Handler: import_tasks my_second_tasks.yml
name[missing]: All tasks should be named.
main.yml:4 Task/Handler: import_tasks my_third_tasks.yml
```
Adding names to the above needlessly clutters the file and obscures my intent.  
According to the docs, I can disable `name[missing]` messages for all tasks/modules: https://ansible-lint.readthedocs.io/rules/name/, **but I still want to require names for tasks which use other modules, just not for the `import_tasks` module**.

Further, when I explicitly say that I don't want a specific module to be used with `name`, I want a rule to enforce that it is never used for that module. I.e., It should be a linting error to use `name` with `ansible.builtin.import_tasks` when I explicitly disable `name` for all uses of the `ansible.builtin.import_tasks` module.

Finally, I also anticipate situtations in which I write my own modules which are similarly self-descriptive (and for which I don't want to use a `name` field). So I'm not suggesting a hard-coded exception for the `ansible.builtin.import_tasks` module specifically. I just want to be able to enforce the absence/presence of the `name` field on a per-module basis for any specified module.
Thoughts?

**Comment by maxmureev** [comment]
I agree with the author of the topic and propose to combine all these modules into requested rule:
- import_tasks/include_tasks
- import_role/include_role
- import_playbook
